src/js/cards.js

```diff
@@ -88,8 +87,7 @@ export const CARDS = {
     permanent: false,
     target: TARGETS.ANY_PERMANENT,
     targetCount: TARGETS.ALL_PERMANENTS,
-    description: "Destroy all cards in play.",
-    flavor: "Watch that basket.",
+    description: ["Destroy all cards in play.", "Watch that basket."],
```

Comment:
I think description will eventually be generated in code, from the cards abilities/keywords, while flavor will just be text written by a human with no game meaning. 

How about removing this change?